System/Constants.h

```diff
@@ -1,6 +1,8 @@
 #ifndef _RTECONSTANTS_
 #define _RTECONSTANTS_
 
+#include "System/Semver200/semver200.h"
```

Comment:
I honestly expected this to be an issue, given that it's such a core file and only has elementary types atm. I can always create a wrapper class/manager to be included, and it can probably contain some of the comparison logic too (caret notation stuff that the base class misses) 

Any suggestions/ preferences would be great, you or @garethyr  etc

Reply:
I think it's pretty much fine if it's just thrown into a System/Versioning.h file along with the game version constant. It doesn't need to be some complex refactor, it's just we should only include these templates where it's strictly necessary.

Reply:
That works for me

Reply:
Resolved in 5372b7ddcea3e886c9c4252699d6cc3969553d9b